### DISCUSSION TITLE
How to check package version installed ?

### DISCUSSION THREAD

**Discussion by zakissimo** [williamboman/nvim-lsp-installer#524] [Q&A] [answered]
I'm having an issue with pyright that is supposedly [fixed since last year](https://github.com/microsoft/pyright/issues/1574)
And I'm wondering if my version is not up to date or I should open an issue ?
How can I easily check my pyright version installed by LspInstaller ?

```
class Foo:
    def __init__(self):
        self.color = None

one = Foo()
one.color = "blue"

Pyright: Cannot assign member "color" for type "Foo"
  Expression of type "Literal['blue']" cannot be assigned to member "color" of class "Foo"
  Type cannot be assigned to type "None"
```

**Comment by williamboman** [comment, marked as answer]
Hello! So I actually just recently added a feature that shows the installed version of a server (see video below). As for your issue, this plugin didn't really exist by the time their fix was released, so I don't believe you're running such an old version. Have you tried simply updating/reinstalling pyright (`:LspInstall pyright`)?

https://user-images.githubusercontent.com/6705160/156441021-62d049e1-6199-4db1-b286-fafa24726dd7.mov

**Reply by zakissimo** [reply]
I updated and now see the version, thanks for the quick answer, I'll open an issue on the pyright github.
Thanks again !
